src/renderer/VersionUpdateWidget.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+const ipc = window.electron.ipcRenderer;
+
+export default function VersionUpdateWidget() {
+
+  const openReleaseDownloadUrl = () => {
+    ipc.sendMessage('openURL', [downloadUrl]);
+  };
+
+  const [widgetClass, setWidgetClass] = React.useState("hidden");
+  const [downloadUrl, setDownloadUrl] = React.useState<string>("");
+
+  React.useEffect(() => {
+    window.electron.ipcRenderer.on('updateAvailable', (downloadUrl) => {
```

Comment:
You have `window.electron.ipcRenderer` aliased above as `ipc` already, might as well use it :)